src/search/components/RealTimeKeyword/RealTimeKeyword.tsx

```diff
@@ -89,18 +68,16 @@ export const RealTimeKeyword = () => {
             <br />
             검색한 키워드
           </StyledHeaderTitle>
-          {realTime ? (
-            <StyledHeaderTime>
-              {`${realTime.year}.${realTime.month}.${realTime.day} ${realTime.hour}:${realTime.min} 기준`}
-            </StyledHeaderTime>
-          ) : null}
+          {isLoading && <StyledHeaderTime>연결 중입니다</StyledHeaderTime>}
+          {data && <StyledHeaderTime>{`${realTime} 기준`}</StyledHeaderTime>}
```

Comment:
```suggestion
<StyledHeaderTime>
  {isLoading ? '연결 중입니다' : `${realTime} 기준`}
</StyledHeaderTime>
```

리액트 쿼리를 안써봐서 잘 모르겠지만 `isLoading`이 `false`일 때 `data`가 존재한다는 것이 보장되면 이렇게 리팩토링할 수 있을 것 같네요

Reply:
[useQuery](https://tanstack.com/query/latest/docs/framework/react/reference/useQuery) 문서 보면 
`isSuccess`가 `true`면 데이터 보여줄 준비가 됐다는 뜻이라고 나와있네요!
![image](https://github.com/yourssu/Soomsil-Web/assets/87255462/a75ba397-bfe5-473f-b8b5-bbcd4fa7d440)

쭌이 제안한 코드에서 `isSuccess`를 써도 좋을 거 같아요 ㅎㅅㅎ

Reply:
헉 공식문서 찾아 볼 생각을 못했네요..

대 보 리

Reply:
감사합니다!_!